backends/advanced/src/advanced_omi_backend/routers/modules/audio_routes.py

```
@@ -106,17 +107,60 @@ async def get_conversation_audio(
+            content=range_data,
+            status_code=206,
+            media_type="audio/wav",
+            headers={
+                "Content-Range": f"bytes {range_start}-{range_end}/{file_size}",
+                "Content-Length": str(content_length),
+                "Accept-Ranges": "bytes",
+                "Content-Disposition": f"inline; filename={conversation_id}.wav",
+                "X-Audio-Source": "mongodb-chunks",
+            }
+        )
+    except (ValueError, IndexError) as e:
```

Ruff (0.14.11)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)